README.md

```diff
@@ -10,6 +10,7 @@
 
 _Primitives are designed for deployment in adversarial environments. If you find an exploit, please refer to our [security policy](./SECURITY.md) before disclosing it publicly (an exploit may equip a malicious party to attack users of a primitive)._
 
+* [broadcast](./broadcast/README.md): Pre-consensus reliable broadcast.
```

Comment:
nit: I think we should remove `pre-consensus` as a descriptor here. We could use these primitives before/during/after consensus and shouldn't scope them to a single phase (or that they even need to work with consensus).